x/proof/module/query_claim.go

```diff
@@ -70,7 +70,7 @@ $ poktrolld q claim list-claims --supplier-address <supplier_address> --node $(P
 
 func CmdShowClaim() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "show-claim <session_id> <supplier_addr>",
+		Use:   "show-claim <session_id> <supplier_operator_addr>",
```

Comment:
Can you add a `TODO_MAINNET: Move from Cobra to AutoCLI for all of these commands` please?

Reply:
There's already `TODO_TECHDEBT`s in `x/<module>/module/tx.go which refer to all the `CmdXXX()`

`// TODO_TECHDEBT(@bryanchriswhite, #370): remove if custom query commands are consolidated into AutoCLI.`
 
Will change them to `TODO_MAINNET`